CRUD.c

```diff
@@ -0,0 +1,209 @@
+#include <stdio.h>
+#include <stdlib.h>
+#include <string.h>
+
+typedef struct{
+    int id;
+    char name[50];
+    int age;
+}User;
+
+void create(FILE *file){
+    User user;
+    int isUnique = 0;
+
+    while(!isUnique){
+        isUnique = 1;
+
+        printf("Enter User_ID: ");
+        if(scanf("%d", &user.id) != 1){
+            printf("Invalid input. Please enter a valid number.\n");
+            while(getchar() != '\n');
+            isUnique = 0;

```

Comment:
I have made use of isUnique variable for two purposes to check valid input as well as unique input for the records instead of defining two different variables. I will redo the assignment with 2 different variables and raise a PR.